assets/controllers/disaffiliate_user_controller.js

```diff
@@ -0,0 +1,48 @@
+import {Controller} from '@hotwired/stimulus';
+import {axiosInstance} from '../js/helpers/axios';
+import Swal from "sweetalert2";
+
+export default class extends Controller {
```

Comment:
Tu peux changer le nom du controller je pense, pour qqch de plus générique, on pourra le réutiliser ailleurs

Reply:
We already have a generic alert-controller (displays alert and replace location on chosen route if validation button is clicked).
This controller display alert and make ajax call to unfollow a beneficiary, also it's a one way event (data action is removed from the button after confirm). This responds to a particular use case